README.md

```diff
@@ -12,10 +12,10 @@ Stable versions of the spec are done via git tags and are shown as releases here
 https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/releases
 
 HTML and PDF versions for stable versions are available via the version selector
-in the lower left on ReadTheDocs. As an example, for version 0.2:
+in the lower left on ReadTheDocs. As an example, for version 0.3:
 
-- HTML at <https://gamma-astro-data-formats.readthedocs.io/en/v0.2/>
-- PDF at <https://media.readthedocs.org/pdf/gamma-astro-data-formats/v0.2/gamma-astro-data-formats.pdf>
+- HTML at <https://gamma-astro-data-formats.readthedocs.io/en/v0.3/>
+- PDF at <https://media.readthedocs.org/pdf/gamma-astro-data-formats/v0.3/gamma-astro-data-formats.pdf>
 
 For v0.2 we also archived the sources of the spec as well as a rendered PDF and HTML version here:
```

Comment:
The doi can only be updated after we made the release, right? Or was it uploaded manually? In that case we could pre-register a DOI with zenodo and already include it here.

@lmohrmann @cdeil ?

Reply:
Can't we just always link to the latest release here? Or put a badge instead, the link would be https://zenodo.org/record/1409830

Reply:
Ok, I just realize the purpose of the entry in the README is different...